### ISSUE TITLE
create in batch caused error

### ISSUE THREAD

**Issue by suddenDisappear** [go-gorm/clickhouse#80]
## Your Question

When i use createInBatch like `global.GVA_CH.Model(v.Interface).CreateInBatch(datas,1000)` OR specified BatchSize,an error will occured.
`io.EOF` error if `gormConfig.SkipDefaultTransaction` set to true and `unpexted packet` error if  `gormConfig.SkipDefaultTransaction` set to false.

## Expected answer

How can i use CreateInBatch in proper way.

**Comment by unownone** [comment]
Yes this seems to be the issue for us too. It seems that as long as your current `len(passed_data) <= batchSize` you are good. But when the `batchSize` is smaller than the passed data, ie logically when more than 1 batch is to be created , the code fails . I feel it's due to the internal logic somewhere where we do not properly parse/ handle batching for clickhouse!
